config.py

```diff
@@ -1421,8 +1421,106 @@ class ArbitraryContrast(TypedDict):
 Whether to subtract the evoked signal (averaged across all epochs) from the
 epochs before passing them to time-frequency analysis. Set this to `True` to
 highlight induced activity.
+
+Note: Note
+    This also applies to CSP analysis.
+"""
+
+###############################################################################
+# TIME-FREQUENCY CSP
+# ------------------
+
+decoding_csp: bool = False
+"""
+Whether to run decoding via Common Spatial Patterns (CSP) analysis on the
+time-frequency representation of the data.
+"""
+
+decoding_csp_times: Optional[ArrayLike] = np.linspace(
+    max(0, epochs_tmin),
+    epochs_tmax,
+    num=6
+)
+"""
+The edges of the time bins to use for CSP decoding.
+Must contain at least two elements. By default, 4 equally-spaced bins are
+created across the non-negative time range of the epochs.
+
+All specified time points must be contained in the epochs interval.
+
+If `None`, do not perform time-frequency analysis, and only run CSP on
+frequency data.
+
+???+ example "Example"
+    Create 3 equidistant time bins (0–0.2, 0.2–0.4, 0.4–0.6 sec):
+    ```python
+    decoding_csp_times = np.linspace(start=0, stop=0.6, num=4)
+    ```
+    Create 2 time bins of different durations (0–0.4, 0.4–0.6 sec):
+    ```python
+    decoding_csp_times = [0, 0.4, 0.6]
+    ```
+"""
+
+decoding_csp_freqs: Dict[str, ArrayLike] = {
+    'custom': [
+        time_frequency_freq_min,
+        (time_frequency_freq_max - time_frequency_freq_min) / 2 + time_frequency_freq_min,  # noqa: E501
```

Comment:
```suggestion
        (time_frequency_freq_max + time_frequency_freq_min) / 2,  # noqa: E501
```
but just use `np.linspace(time_frequency_freq_min, time_frequency_freq_max, 3)`